BoxContentSDK/BoxContentSDK/External/KeychainItemWrapper/BOXKeychainItemWrapper.m

```diff
@@ -342,6 +343,11 @@ - (void)writeToKeychain
         
         result = SecItemUpdate((CFDictionaryRef)updateItem, (CFDictionaryRef)tempCheck);
         BOXAssert( result == noErr, @"Couldn't update the Keychain Item with error %d.", (int)result);
+        NSDictionary *userInfo = @{@"completion_status" : (result == noErr) ? @"succeeded" : @"failed",
+                                   @"message" : @(result)};
+        [[NSNotificationCenter defaultCenter] postNotificationName:BOXRefreshTokenSaveToKeychainNotification
```

Comment:
in Toolkit SDK, the keychain item could be anything, and not just refresh token. We should make setObject:forKey: to return result code. And send notification from BOXContentSDK's storeCredentialsToKeychain where we call setObject to persist session credentials including user info and tokens.